src/Service/FileCache.php

```diff
@@ -0,0 +1,110 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Service;
+
+/**
+ * This class manages cache for files being translated
+ */
+class FileCache
```

Comment:
Could this extend `\Symfony\Component\Cache\Adapter\FilesystemAdapter`? That way we don't have to re-invent things like expiry tracking and we'd be able to e.g. prune expired items via the CLI.

Reply:
That's what I started from, however I was frustrated by inability to access the cache file directly, which would make difference when loading huge files.

Reply:
Ah sure, sounds sensible.